src/app/searchResult.ts

```diff
@@ -0,0 +1,6 @@
+import {Video} from "./video";
+
+export interface SearchResult {
```

Comment:
you need to fully describe the interface of the response model that is provided to you in the assignment